frontend/packages/core/src/AppProvider/short-link.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import type { clutch as IClutch } from "@clutch-sh/api";
+
+import { useNavigate } from "../navigation";
+import { client } from "../Network";
+import type { ClutchError } from "../Network/errors";
+
+interface ShortLinkProps {
+  hydrate: (data: IClutch.shortlink.v1.IShareableState[]) => void;
+  onError: (error: string) => void;
```

Comment:
Should we be using `ClutchError` here instead?

Reply:
Originally it was because I was setting a string for the toast but I can return the error and let the parent handle it